src/main/java/frc/trigon/robot/subsystems/coralintake/CoralIntakeConstants.java

```diff
@@ -229,64 +220,53 @@ private static void configureMasterAngleMotor() {
         config.HardwareLimitSwitch.ReverseLimitEnable = true;
         config.HardwareLimitSwitch.ReverseLimitType = REVERSE_LIMIT_TYPE_VALUE;
         config.HardwareLimitSwitch.ReverseLimitSource = REVERSE_LIMIT_SOURCE_VALUE;
+        config.HardwareLimitSwitch.ReverseLimitAutosetPositionEnable = true;
+        config.HardwareLimitSwitch.ReverseLimitAutosetPositionValue = ELEVATOR_REVERSE_LIMIT_POSITION_ROTATIONS;
```

Comment:
Sorry for the headache, the autoset should be forward just because it's easier that way mechanically